python/fledge/services/core/server.py

```diff
@@ -1159,6 +1159,8 @@ async def register(cls, request):
                     if not cls._storage_client_async is None:
                         cls._audit = AuditLogger(cls._storage_client_async)
                         await cls._audit.information('SRVRG', {'name': service_name})
+                        await ACLManager(cls._storage_client_async).\
```

Comment:
really? 

Would not it read from security Configuration?

I was assuming we only care for Unresponsive one and that will be with Monitoring loop